post-processing/fuzz_html.py

```diff
@@ -1236,6 +1278,7 @@ def create_html_report(
     # Section with details about each fuzzer, including calltree.
     #############################################
     logger.info(" - Creating section with details about each fuzzer")
+    fuzzer_table_data = typing.Dict[str, Any]
```

Comment:
This should be `fuzzer_table_data: typing.Dict[str, Any] = dict()`

Reply:
Fixed in https://github.com/ossf/fuzz-introspector/commit/b4f6e9d373a87460bfb2637c4dabb140644662b4